### DISCUSSION TITLE
在openwrt中使用夸克网盘，会经常自动停止运行

### DISCUSSION THREAD

**Discussion by zxkke** [AlistGo/alist#5713] [Q&A]
在openwrt中使用夸克网盘，会经常自动停止运行，其他平台没有测试

**Comment by IAALAI** [comment]
你有没有尝试检查你的设备的状态？内存之类的是否正常？

**Reply by zxkke** [reply]
以前从来没有过这种现象，openwrt和docker都使用过，在使用夸克网盘播放一段时间视频就会自动停止，但是docker会自动重新启动alist

**Comment by krasie** [comment]
同遇到这个问题，在 windows 是正常，就是 openwrt 有问题
